### PR TITLE
Feature/bump minsdk to ios 9

### PR DESCRIPTION
**Pull request by TroubleMakerBen**
This contains required code changes. We have a separate work item for the documentation updates. I've also added the #dontmerge label as we're still waiting on feedback related to the version.

### CODE REVIEW

File: TestApps/Sasquatch/Sasquatch/ViewControllers/MSCrashesViewController.swift [hunk 1]
```diff
@@ -76,7 +76,7 @@ class MSCrashesViewController: UITableViewController, UIImagePickerControllerDel
       } else if (indexPath.row == 1) {
         cell.textLabel?.text = "Text Attachment";
         let text = UserDefaults.standard.string(forKey: "textAttachment")
-        cell.detailTextLabel?.text = (text?.characters.count ?? 0) > 0 ? text : "Empty";
+        cell.detailTextLabel?.text = (text?.count ?? 0) > 0 ? text : "Empty";
```
Comment (MatkovIvan) [inline TestApps/Sasquatch/Sasquatch/ViewControllers/MSCrashesViewController.swift]: This is the diff between Swift versions. And as far as I know, we are still using XCode 8. So, that is why CI build was failed.

Comment (TroubleMakerBen) [reply TestApps/Sasquatch/Sasquatch/ViewControllers/MSCrashesViewController.swift]: right...reverting this now.

**Comment by TroubleMakerBen** [comment]
@jaelim-ms Yes, there is a PR for our documentation that is already approved and will be merged once the SDK has been released.

File: Puppet/Puppet.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1307,7 +1307,7 @@
 				DEVELOPMENT_TEAM = "";
 				GCC_OPTIMIZATION_LEVEL = 0;
 				INFOPLIST_FILE = NotifExtension/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (MatkovIvan) [inline Puppet/Puppet.xcodeproj/project.pbxproj]: Notification extension is iOS 10+ functionality

Comment (TroubleMakerBen) [reply Puppet/Puppet.xcodeproj/project.pbxproj]: Damn! How did that happen.
Thx for the fix Ivan!

**Pull request merged**